examples/restrict-by-media-type/restrict_by_media_type.client.out

```diff
@@ -8,8 +8,8 @@ $ curl -v http://localhost:9092/infoService -H "Accept:application/xml" -H "Cont
 * Connection #0 to host localhost left intact
 <name>Ballerina</name>
 
-# Use the following client to invoke the service using unsupported media type. The Content type of the
-# request is not listed under consumes resource configuration.
+# To invoke the service using unsupported media type, execute the following client. The Content type of the
```

Comment:
```suggestion
# To invoke the service using unsupported media type, use the following cURL request. The content type of the
```

Reply:
The word "execute" is more relevant than "use" here since the context is about a cURL command.

Reply:
Changed to "To invoke the service using unsupported media type, execute the following cURL request. The content type of the request is not listed under the consumer's resource configuration.".